h26x_extractor/__main__.py

```diff
@@ -9,21 +9,47 @@
 
 Example:
   h26x-extractor -v file1.264 file2.264
-"""
 
-from docopt import docopt
+Install from sources: 
+  python .\setup.py install
+  python -m h26x_extractor --version
+if a build directory exists, remove it with:
+  rm build
+  rm dist
+  rm h26x_extractor.egg-info
+After installation, you can run the program with:
+  h26x-extractor -v file1.264 file2.264
 
+"""
+import sys, time
+from docopt import docopt
 from . import __version__
-
 from . import h26x_parser
 
+def timeit(method):
+    def timed(*args, **kw):
+        ts = time.time()
+        result = method(*args, **kw)
+        te = time.time()
+        print('%r  %2.2f ms' % \
+                (method.__name__, (te - ts) * 1000))
+        return result
+    return timed
+
+args = docopt(__doc__, version=str(__version__), options_first=False)
 
 def main():
-    args = docopt(__doc__, version=str(__version__), options_first=False)
     for f in args["<input-file>"]:
         ex = h26x_parser.H26xParser(f, verbose=args["--verbose"])
         ex.parse()
 
-
 if __name__ == "__main__":
-    main()
+    if args["--verbose"] == True:
+      timeit(main)()
+    else:
+      main()
+    if args["--verbose"] == True:
+      print('Verbose is true')
+    else:
+      print('Verbose is false')
+

```

Comment:
This looks a bit clunky to me. Note that `if x == True` is not pythonic. Simply use `if x`.

```python
start = timeit.default_timer()
main()
stop = timeit.default_timer()
if args["--verbose"]:
    print('Time: ', stop - start)  
```

Reply:
Sorry, python is not really my first language 😞 
Changed it.